src/Command/FindAndUploadFilesCommand.php

```diff
@@ -171,6 +172,12 @@ protected function configure(): void
                 null,
                 InputOption::VALUE_OPTIONAL,
                 'Default branch for the repository'
+            )
+            ->addOption(
+                self::OPTION_DISABLE_CONDITIONAL_SKIP_SCAN,
+                null,
+                InputOption::VALUE_NONE,
+                'Use this option to disable skip scan from ever triggering. Default is to allow skip scan triggering because of long queue times (=false).'
```

Comment:
Surely this command doesn't need this option?

Reply:
Unfortunately yes because compoundcommand inherits from this command. And it is compoundcommand we run from debricked-scan. I tried adding it to just compoundcommand but it said it could not find the option.

Reply:
ah, that's a bit unexpected

Reply:
It happens when we try to run findandupload in compound, we just pass the input from compund to findandupload and it can't find the option unless it is defined in findandupload. I think this would be the cleanest solution, otherwise we have to copy over all options except conditional_skip_scan option to a new ArrayInput and pass that to findandupload.

Reply:
`$findAndUploadReturnCode = $findAndUploadCommand->run($input, $findAndUploadOutput);`, line 73 in CompoundCommand.php

Reply:
unset the option when passing it to $findAndUploadCommand->run

Reply:
Here is a solution, but I don't like it 9746df6bff5f4e9ee05856135d4117a231ce2c07 :smile:

Reply:
it is a bit ugly, but the separation of concerns is better